experimental/telemetry/rudder.go

```diff
@@ -0,0 +1,46 @@
+package telemetry
+
+import rudder "github.com/rudderlabs/analytics-go"
+
+const rudderDataPlaneURL = "https://pdat.matterlytics.com"
+
+var rudderWriteKey string
```

Comment:
@larkox, can you add some GoDoc to these files to help clarify intent and usage? I have a bunch of comments, but don't want to make assumptions without better understanding how things are meant to be used.